data/human/deep missions.txt

```diff
@@ -2472,28 +2472,48 @@ mission "Stone of our Fathers 1"
 			`Helheim's spaceport has all the constant hustle and bustle of a boomtown. It's dirty, lively, and packed with workers. As you walk through the spaceport, an aged man in a worn and weathered roustabout jacket waves his arm toward you and approaches you.`
 			`	"You've got the look of one who knows a ship inside and out," he says. "Name's Skaldgar, ain't much for ships; earth, soil, and rock are my domain. But I do got a need to travel." He pauses as a wistful look briefly passes over his face. "I left, you see, decades ago. No goodbye. No note. Just... left. Living with others, taking care of others, hard on a man like me. I like solitude, peace, and work. But now my bones are brittle: the radiation, earth-rot as we call it, has taken hold and my time is running out. And I find, I don't want to be alone anymore."`
 			`	He stops and frowns. "Look at me, rambling like an ol' man. Ser, Captain, I'd like to pay you <payment> to take me home, to <destination>."`
+			label choice
 			choice
 				`	"If you have coin, I have room."`
 				`	"I guess I can take a small detour."`
+				`	"Why are you calling me 'ser'?"`
+					goto ser
+					to display
+						not "Stone of our Fathers: ser"
 				`	(Shake my head. I have better things to do than bus old men around.)`
 					goto decline
+
+			action
+				clear "Stone of our Fathers: ser"
 			`	He nods, grunts, and heads to your ship with a small knapsack cradled under his arm, clearly ready to travel to <planet> at a moment's notice.`
 				accept
+
+			label ser
+			action
+				set "Stone of our Fathers: ser"
+			`	"You're not from around here, are ya? Ser's just what decent folk call each other in the Deep. Sign of respect and the like. Though I am hearing it less and less these days. Old ways being forgotten, but ain't that the way of things?"`
```

Comment:
```suggestion 
			`	"You're not from around here, are ya? Ser's just what decent folk call each other in the Deep, man or woman. Sign of respect and the like. Though I am hearing it less and less these days. Old ways being forgotten, but ain't that the way of things?"`
```

Perhaps something like this would work to highlight it is explicitedly gender neutral? 

I really don't get the impression that that implies that as it stands, it just feels like they're explaining why they're calling me sir, albeit the Deep's accented version of it.

Reply:
I find this problematic. 

First, by saying it applies to men or women you are making a phrase that is non-binary; binary. Feels like a step backwards. 

Second, I'd argue that Skaldgar not making it gendered is enough to make it clear it isn't gendered. You've asked for an explanation and he is clearly not saying it's just for men.  It's explicit in the omission. There is something refreshing in him not even viewing the question through a gendered/binary lens. It's just a nice thing that folks call each other. 

Third, even if I were to render this in language that feels a little more like Skaldgar, and is not binary... for instance...
"...folk call each other in the Deep, regardless of whether their gals, guys, or anything else." 

It just feels off. It feels explicit and immersion breaking. Like he is addressing the person behind the keyboard and not the character in the game.

Reply:
This sounds like a more elegant solution.

Reply:
Yes, a follow up question works best! The binary nature of that suggested sentence is the reason i said "_something like this_" as opposed to simply "this". Thanks, @SpearDane!

I wasn't satisfied with it, but <ins>something</ins> was absolutely necessary. Omitting if it is or isn't gender neutral absolutely does not imply it gender neutral if the questioner believes it is a clearly gendered word, which is exactly what lead to the added text string in the first place! 

If an until then non-gendered player character questions why they are being called **Sir** and is told it is "just what decent folk call each other", it _sounds_ like it confirms one's a man, or at least male presenting, to the eyes of the NPC. "Why yes, sir _is_ what decent folks call each other... if they are men. Sounds like i'm one, then." is my worry of what would go through many player's minds reading only the original question.

This follow-up works to fix that without making the old men immediately jump to a gendered lens - great suggestion, Dane! 🎉

### Here's an additional suggestion, though:

What about "So Ser is different from Sir?" as an <ins>alternative</ins> [2] option to the follow-up question, with an edited answer that acknowledges the reasonableness of the player's confusion?

It'd be for flavour, seeing as the implemented option already makes it sufficiently explicit, but while implying the player is foolish for asking - "What does that have to do with anything?" with a confused expression really doesn't go a long way to make the person behind the keyboard feel heard through the player character, and may itself break immersion. That is never good as the only option <ins>unless it is meant to be foolish to ask, or at least asked in a foolish way</ins>. I understand this is highly unlikely to be what you meant by it, but i find it'll come off that way to some! 

It does succeed at implying that the old man doesn't see things through that gendered lens, and that is really great, but it really can _feel_ more like it's telling the person behind the keyboard "it's gender neutral, you #%@%, why would it _possibly_ not be?" as opposed to "oh no no, Sir is a different word to Ser, ser!". 

It isn't an unreasonable confusion at all and it is never desirable to imply the player is unwise for being confused with writing which is ambiguous. "So Ser is different from Sir?" also makes the person behind the keyboard's confusion **very explicitly** the same as First-Last's confusion, avoiding speaking directly to the human player as opposed to the in-game character when tackling it.

So, what about this as a second option to the follow-up:

[1]"Does Ser just apply to men?"
[2] "So Ser is different from Sir?"
The old man's eyes widen slightly before he nods and says, "Oh, oh yes, i see your confusion now. What's the bits between a fella's legs really has not to do with anything. Like I already said, 'Ser' is just what decent folk call each other; ain't nothing more to it. Don't matter if you're a guy, gal, or anything else. Hel, I reckon I'd even call an elf 'Ser' if they done me a good turn."

With that, i find that it enhances option [1] in turn by making the slightly ruder answer there be due to your unwise choice of words!

What do ya'll think?

Reply:
Both versions are fine but I think I prefer @LorenzoBolla's  a little more. It feels a little friendlier.

Reply:
Eh. 

I'd like to point out that I've read books that used 'Ser', and never once did they feel the need to go out of their way to explicitly explain its use. The authors assumed their readers were smart enough to figure it out via context, and I think we're being a little overly cautious already.  

Also, it's important to remember that this scene being spoken, and saying "So Ser is different from Sir?" sounds ridiculous and senseless when spoken out loud. 

Skaldgar behaves the way he does above because he believes it is a foolish thing to be asking him. He already humored you by answering your initial question, to his mind quite satisfactorily, despite it having no baring on anything he needs from you or his mission with you. Asking it again does annoy an old codger like him, hence his response.

Reply:
@SpearDane 99% of the books that use Ser that I've read use it as a "sir" equivalent (As in, an exact equivalent, being fantasy and/or pseudo medieval, women are ladies and men are sers)

Reply:
I saw ser used this way in some other fantasy/pseudo-medieval books, not just GRRM's.

Don't know about sci-fi though

Reply:
Fair enough anyway : )

Reply:
> I'd like to point out that I've read books that used 'Ser' and never once did they feel the need to go out of their way to explicitly explain its use. I've never read a character/narrator define it for the audience. The authors assumed their readers were smart enough to figure it out via context. 

Considering that we were unable to figure it out via context, to the point it was removed from the game on #9622, the same cannot be done here. It was tried, and now 2 PRs were the result. Perhaps it's a difference in audience.

I strongly believe it would be very unwise to add it back in without making it explicit, as it does not fix the issue which started this to begin with. It's not a may-as-well-while-adding-it-back-in, but a prerequisite.

> Also, it's important to remember that this scene is being spoken, and saying "So Ser is different from Sir?" sounds ridiculous and senseless when spoken out loud.

I assumed that Ser was pronounced differently than Sir; is it _really_ pronounced the _exact_ same, with no phonetic difference <ins>at all</ins>? 

The whole premise is that people think that Ser is Sir with an accent which pronounces it with an E, same as the W to V accent seen in the same mission string.

I had personally read it as pronounced like the "ser" seen in my language (the verb "to be" in the infinitive and the word "being" as in "human being"; "ser humano", in Portuguese). I don't have enough knowledge of the phonetic alphabet to explain it better.

"So Ser is different from Sir?" reads, to someone thinking that, as First-Last explicitly pronouncing Ser with an E, and Sir with an I, pointing out their (and the human player's) confusion very clearly and concisely. To be unable to do that, with the option which _is_ made available leading to a confused, coarse answer from the NPC, would lead to a frustrating situation for the player. 

"Oh, I'd have asked that better, come on!" Is not the desired sensation we wish to transmit, and it does not to a good job at maintaining immersion. Really, it completely shatters it. Even if Ser _actually_ sounds exactly like Sir, the human player does not think that and therefore such a scenario has to be avoided entirely.

If it is pronounced the same, then it cannot be translated as First-Last's confusion as it doesn't sound that way to them, in which case it is quite problematic to use it as it leads to confusion unexplainable without speaking to the human behind the keyboard or otherwise frustrating them. And remember, it clearly requires explanation for the majority for the Endless Sky player base.

Reply:
The more times I come back and read the new versions the less I like it. I don't really like the Skaldgar response that is given when when asked for an explanation. It makes me think " Gee, sorry for asking"

Reply:
> Is it _really_ pronounced the _exact_ same, with no phonetic difference at all?
> 
> I had personally read it as pronounced like the "ser" seen in my language (the verb "to be" in the infinitive and the word "being" as in "human being"; "ser humano", in Portuguese). I don't have enough knowledge of the phonetic alphabet to explain it better.

Fair enough. If you feel that way then I see no reason to remove it as a possible response/question.